mapbox/scripts/geocoder.py

```diff
@@ -8,18 +8,42 @@
 class MapboxException(click.ClickException):
     pass
 
+def _is_numeric(x):
+    try:
+        float(x)
+    except:
+        return False
+    return True
 
 @click.command(short_help="Geocode an address.")
-@click.argument('ADDRESS')
 @click.option('--access-token', help="Your access token")
+@click.option('--forward', default=False, help="Perform a forward geocode")
+@click.option('--reverse', default=False, help="Perform a reverse geocode")
```

Comment:
We can combine these into one [boolean option](http://click.pocoo.org/4/options/#boolean-flags) like

`@click.option('--forward/--reverse', default=True, help="Perform a forward (default) or reverse geocode")`

 and remove lines 29-33.